backend/fms_core/resources/sample.py

```diff
@@ -175,7 +175,25 @@ def import_obj(self, obj, data, dry_run):
         except Exception as e:
             individual_created = False
             individual = None
-            errors["individual"] = ValidationError(e.messages.pop(), code="invalid")
+            errors["individual"] = ValidationError("Individual could not be processed due to missing or invalid information.", code="invalid")
```

Comment:
This message is a bit vague to me, but if getting something more specific is much more complex, I would keep it that way.